docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Docker Compose file for FreeCodeCamp
+#
+# Bootstrap with:
+#   docker-compose run --rm server npm install
+#   docker-compose run --rm server npm run only-once
+#
+# Run with:
+#   docker-compose up
+#
+version: "3"
+services:
+    db:
+        image: mongo:3.2.6
+        ports:
+            - "27017:27017" # Change port if needed
+    freecodecamp:
+        image: node:8.9.4
```

Comment:
I'd still suggest pulling something like  `freecodecamp/node:stable` but that is for another PR.